docs/contributing/opensauced-writers-guide/how-to-add-your-post.md

```diff
@@ -0,0 +1,98 @@
+---
+id: writers-guide
+title: "OpenSauced Writers Guide"
+sidebar_label: "OpenSauced Writers Guide"
+keywords: 
+- "writers guide" 
+- "open source writers guide" 
+- "open source writer" 
+---
+
+Welcome to our Writers Contributor Guide. This will walk you through the steps of how to add your blog post to our community resources using the docs repository.
+
+## Repeat Authors: Adding to `authors.yml`
+
+For regular blog post authors, we should declare those authors globally in the configuration file:
+
+**Location:** `/blog/authors.yml`
+
+**Example:**
+```yaml
+BekahHW:
+  name: Bekah Hawrot Weigel
+  title: Developer Experience Lead
+  url: https://twitter.com/BekahHW
+  image_url: https://github.com/BekahHW.png
+  email: Bekah@opensauced.pizza
+  twitter: BekahHW
```

Comment:
[proofreading/suggestion]: I think it would be helpful to name it x(twitter) for accuracy. 
```suggestion
 x(twitter): BekahHW
```

Reply:
That's not the docusaurus format. They specifically use Twitter.

Reply:
Oh ok @BekahHW.